ephios/core/models/users.py

```diff
@@ -249,6 +277,8 @@ class QualificationGrant(Model):
     )
     expires = models.DateTimeField(_("expiration date"), blank=True, null=True)
```

Comment:
or we could just use a datefield instead?

Reply:
I think we chose a datetimefield because it provides an exact point in time when the expiration happens, rather than being ambiguous on the data level.